yb-voyager/cmd/exportSchema.go

```diff
@@ -72,12 +73,19 @@ func exportSchema() {
 		utils.ErrExit("Failed to connect to the source db: %s", err)
 	}
 	source.DB().CheckRequiredToolsAreInstalled()
+	sourceDBVersion := source.DB().GetVersion()
 
-	fmt.Printf("%s version: %s\n", source.DBType, source.DB().GetVersion())
+	fmt.Printf("%s version: %s\n", source.DBType, sourceDBVersion)
 
 	CreateMigrationProjectIfNotExists(source.DBType, exportDir)
 	source.DB().ExportSchema(exportDir)
 	utils.PrintAndLog("\nExported schema files created under directory: %s\n", exportDir+"/schema")
+
+	callhome.InitJSON(exportDir)
```

Comment:
Every occurrence of `callhome.GetPayload()` is preceded by `callhome.InitJSON()`. Can we move the call to the `initJSON()` inside the `GetPayload()`? If the "json" is already initialised, `GetPayload()` will simply skip reinitialising the json.

Reply:
Good catch, I was searching for a global initialisation elsewhere in the code, but we run into the risk that the meta info folder was never created- will add this change 👍🏽